.github/workflows/python-package-conda.yml

```diff
@@ -5,33 +5,39 @@ on: [push]
 jobs:
   build-linux:
     runs-on: ubuntu-latest
-    strategy:
-      max-parallel: 5
```

Comment:
does this still run the process in parallel? Or does it serialize the snakemake runs?

Reply:
I don't think this had the intended effect of parallelising snakemake runs. Removing it shouldn't slow down pytest